### PR TITLE
[ADLA] - Catalog - Add stream path to USqlTableFragment

### PR DESCRIPTION
**Pull request by idear1203**
### Swagger PR Links
- [Azure/azure-rest-api-specs#4676: typo: Microsoft.DataLakeAnalytics](https://github.com/Azure/azure-rest-api-specs/pull/4676)
- [Azure/azure-rest-api-specs#5329: [ADLA] - Catalog - Add stream path to USqlTableFragment](https://github.com/Azure/azure-rest-api-specs/pull/5329)

### Note
- [Azure/azure-rest-api-specs#4676: typo: Microsoft.DataLakeAnalytics](https://github.com/Azure/azure-rest-api-specs/pull/4676)
    * partion -> partition
    * formatxii -> format
    * occured -> occurred
    * underying -> underlying
    * Double word "delete"
Asunc -> Async
- [Azure/azure-rest-api-specs#3013: [ADLA] - Support table preview](https://github.com/Azure/azure-rest-api-specs/pull/3013)
    - Added `StreamPath` of type string to `USqlTableFragment`

### CODE REVIEW

**Review by t-rufang — APPROVED** [review]
LGTM! Ship it!

File: src/SDKs/_metadata/datalake-analytics_data-plane.txt [hunk 1]
```diff
@@ -3,12 +3,12 @@ AutoRest installed successfully.
 Commencing code generation
 Generating CSharp code
 Executing AutoRest command
-cmd.exe /c autorest.cmd https://github.com/Azure/azure-rest-api-specs/blob/master/specification/datalake-analytics/data-plane/readme.md --csharp --version=latest --reflect-api-versions --csharp-sdks-folder=A:\azure-sdk-for-net\src\SDKs
```
Comment (dsgouda) [inline src/SDKs/_metadata/datalake-analytics_data-plane.txt]: Is anything being generated for dataplane in this PR?
If not, please revert this file.

**Review by dsgouda — CHANGES_REQUESTED** [review]
Looks good for the most part

File: src/SDKs/_metadata/datalake-analytics_data-plane.txt (hunk 1 above)
Comment (idear1203) [reply src/SDKs/_metadata/datalake-analytics_data-plane.txt]: Thanks, this is a dataplane related PR. The swagger spec file can be found [here](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/datalake-analytics/data-plane/Microsoft.DataLakeAnalytics/stable/2016-11-01/catalog.json).

Comment (dsgouda) [reply src/SDKs/_metadata/datalake-analytics_data-plane.txt]: I see, Shouldn't be a blocker.

**Review by dsgouda — APPROVED** [review]

**Pull request merged**